docs/userGuide/tipsAndTricks.md

```diff
@@ -34,18 +34,33 @@ For Markdown syntax: To display a literal character that are normally used for M
 
 ##### :fas-lightbulb: Using {% raw %}{% endraw %} to display `{{ content }}`
 
+By default, MarkBind processes any code in the form of `{{ content }}`. This is because Nunjucks (which is [supported by MarkBind](markBindSyntaxOverview.html#support-for-nunjucks)) uses this syntax to [evaluate an expression, variable, or function calls](https://mozilla.github.io/nunjucks/templating.html#variables). For instance:
 
-To display the raw variable interpolation syntax using `{% raw %}{% endraw %}`, you would also need to add
-the `v-pre` attribute using markdown-it-attrs or as a HTML attribute.
+```
+{{ username }}
+```
+
+This will not display `{{ username }}` as a raw string, but instead, look up the variable `username` from the context and display its value.
+
+In general, to use this syntax as a raw string in a code block or a template, there are [two methods](https://jinja.palletsprojects.com/en/3.0.x/templates/#escaping) available. The syntax can either be encased as a variable like `{{ '{{' }}` and `{{ '}}' }}`, or alternatively, the entire code block can be encased with the raw-endraw tags: `{% raw %} {{ content }} {% endraw %}`. 
 
 <box type="info">
 
-This isn't necessary for `<code>` elements, markdown code fences and inline code though, which MarkBind automatically
-adds `v-pre` for.
+If using raw-endraw tags on an HTML element, you would also need to add the `v-pre` attribute to the HTML tag, like this:
 
-However, this does not change the need for `{% raw %}{% endraw %}`. Meaning, you can still use variables within your code!
-</box>
+```
+{% raw %}
+
+<p v-pre>{{ content }}</p>
```

Comment:
Can we change this to `<div></div>` instead, to avoid hydration errors. Also, what about the markdown-it-attrs method previously mentioned? You removed it because it won't work or?

Reply:
Can I confirm that when the `v-pre` attribute is added, compilation is skipped on the Vue side for the element and its children as per the docs [here](https://vuejs.org/api/built-in-directives.html#v-pre), so even if a `<p>` tag is used, hydration errors should not occur since `v-pre` attribute has been added? 

This is just a side question, I'll add in a `<div></div>` to follow conventions :+1: 

Also, as for the `markdown-it-attrs` method, I initially removed it as I thought it was not working with the current implementation, but after some digging and testing, it still seems to work - I will be adding in a link and an example for this as well!